public/common/components/button/view-models/button.viewmodel.coffee

```diff
@@ -0,0 +1,4 @@
+class ButtonViewModel
+	constructor: ({ @text, @click }) ->
```

Comment:
Create own `nx.Cell` and expose it to the outer world instead of passing `@click`